simul/main.go

```diff
@@ -17,11 +17,17 @@ import (
 var configFlag = flag.String("config", "", "TOML encoded config file")
 var platformFlag = flag.String("platform", "", "name of the platform to run on")
 var runTimeout = flag.Duration("run-timeout", 2*time.Minute, "timeout of a given run")
+var pemFile = flag.String("pemFile", "", "location of the .pem file for EC2 ssh")
+var regions = flag.String("regions", "us-west-2", "list of AWS regions")
 
 func main() {
 	flag.Parse()
+
+	parameters := make(map[string]string)
+	parameters["pemFile"] = *pemFile
+	parameters["regions"] = *regions
 	c := lib.LoadConfig(*configFlag)
-	plat := platform.NewPlatform(*platformFlag)
+	plat := platform.NewPlatform(*platformFlag, parameters)
```

Comment:
Sorry I should have documented that earlier. There are two kind of config a platform needs:
1. Config to run the main simulation binary on the laptop -> this snippet. 
2. Config on the `node/main.go` binary: for this one, my idea was that the `platform` package could define itself its flags needed, without having to worry about changing the `main.go`. This way you dont need to pass extra parameters, `aws` define its own flags in its own package (maybe) and after the `flag.Parse`, every needed information is available and `node/ :)
But in the end the same technique could be used for 1 also I guess, but let's say it's fine for now it's not a big deal for me ;)

Reply:
Related to:
https://github.com/ConsenSys/handel/issues/18

Once we start working on the above issue let's come back to this discussion.